workspace/TS100/src/main.cpp

```diff
@@ -596,7 +597,7 @@ static void gui_solderingMode() {
 
 static const char *HEADERS[] = {
 __DATE__, "Heap: ", "HWMG: ", "HWMP: ", "HWMM: ", "Time: ", "Move: ", "Rtip: ",
-		"Ctip: ", "Vin :" };
+		"Ctip: ", "Vin :", "THan: " };
```

Comment:
THan confused me, maybe Ctip and Vin should also be changed to CTip and VIn?

Reply:
I agree that they should be changed to match, I think its more logical with the second upper case too... Though VIn feels weird, but maybe thats just because i see VIN / vin fairly often :/